### DISCUSSION TITLE
WP_Query integration

### DISCUSSION THREAD

**Discussion by sc0ttkclark** [pods-framework/pods#6620] [Ideas]
Hook into WP_Query for meta_query and other references to pod fields, to join and replace those names with the true aliases (for relationships) just like find() does.

**Comment by sc0ttkclark** [comment]
Need to hook into meta_query to take over Pods fields (table-based); When orderby => meta_value we need to hook into meta_key => field_name for Pods fields (table-based)

**Comment by mattbanks** [comment]
Will hooking in to WP_Query allow for displaying the body in pods queries (post_content column) properly passed through all the normal WordPress filters (i.e. oEmbed parsing, wpautop, etc)? That's the biggest thing I would like to see, either just in PHP using display() or field(), or in magic tags inside pods templates.

**Comment by sc0ttkclark** [comment]
Can you add a separate github issue for that? We'll make sure it happens for Pods 2.3

**Comment by mattbanks** [comment]
Sure thing! I think we talked about it on Twitter a couple weeks ago and just remembered, so I was looking through current issues before posting a new one.

**Comment by sc0ttkclark** [comment]
Slightly related, meta_query arguments now accepted in find() etc https://github.com/pods-framework/pods/commit/0ede9a06df173ada6879307a2b7c114963682e86
